frontend/apps/reader/readerui.lua

```diff
@@ -530,9 +530,11 @@ function ReaderUI:registerKeyEvents()
             if Device:hasKeyboard() then
                 self.key_events.KeyToggleWifi = { { "Shift", "Home" }, event = "ToggleWifi" }
                 self.key_events.OpenLastDoc = { { "Shift", "Back" } }
+                self.key_events.ShowBookMap = { { "Shift", "Down" }, event = "ShowBookMap" }
             else -- Currently exclusively targets Kindle 4.
                 self.key_events.KeyToggleWifi = { { "ScreenKB", "Home" }, event = "ToggleWifi" }
                 self.key_events.OpenLastDoc = { { "ScreenKB", "Back" } }
+                self.key_events.ShowBookMap = { { "ScreenKB", "Down" }, event = "ShowBookMap" }
```

Comment:
Logically, these should be in ReaderThumbnail where the event handler is, not because it wouldn't work here, just for consistency with how we do that elsewhere.
But there's currently no event registration in ReaderThumbnail, so dunno if we should keep adding it here to not add noise there. But it looks a bit odd here.

Reply:
so... should I make one there or keep it here?

Reply:
@hius07 @Frenzie : what do you think?

Reply:
I think in ReaderThumbnail.

Reply:
I'd also add them in ReaderThumbnail for clarity.

Reply:
done.